README.md

```diff
@@ -22,7 +22,7 @@ Simultaneously, register yourself on the [Hacktoberfest Website](https://hacktob
 
 1. Create an empty directory in your local system and `cd` into it
 ```bash
-mkdir open_source
+make open_source
```

Comment:
do you really know what mkdir is used for stop spamming do some real fact check.